content/en/synthetics/_index.md

```diff
@@ -8,73 +8,49 @@ further_reading:
-
-* `title` and `message` - text search
-* `status` - Alert, No Data, Ok
-* `creator`
-* `region`
-* `muted`
-* `notification`
-* `tags`
+Synthetic tests allow you to observe how all your systems are performing as experienced by your users from around the globe -  Datadog tracks the performance of your applications and API endpoints using simulated user requests and browser rendering. Synthetics helps you ensure uptime, identify regional issues, compute error budgets, automate tests, and track application performance. By unifying Synthetics with your metrics, traces, and logs, you can test your production environment end-to-end.
 
-To run a search, construct your query using the checkboxes on the left and/or the search bar along the top. The search bar updates with the equivalent query as you check the boxes. Likewise, when you modify the search bar query (or write one from scratch), the checkboxes update to reflect that change. Query results update in real-time as you edit the query; there's no 'Search' button to click.
+## Getting Started
 
-### Write a query
+### Set up API tests
 
-To search for specific text across all checks, titles, and notification messages, enter your text in the search bar.
+Monitor how your customers from around the world experience your http API endpoints using [Synthetics API tests][1].
 
-Otherwise, use Boolean operators (`AND`, `OR`, and `NOT`) and parentheses to write complex queries using any check field:
+{{< img src="synthetics/api.png" alt="API tests"  style="width:100%;">}}
 
-* Regular expressions are not supported
-* Single-character wildcard (`?`) is not supported, but the general wildcard (`*`) is
-* Proximity searches are not supported, but the [fuzzy][1] operator is
-* Ranges are not supported
-* Boosting is not supported
+### Set up Browser tests
 
-Finally, the following characters are reserved: `-`, `(`, `)`, `"`, `~`, `*`, `:`, `.`, and whitespace. To search check fields that include any of them, wrap the field string in quotes: `status:("OK") AND "doc-check"` is a valid query string; `status:("OK") AND doc check` is not.
+Monitor how your customers from around the world experience your webpage using [Synthetics Browser tests][2].
 
-## Create a check
+{{< img src="synthetics/browser.gif" alt="Browser tests"  style="width:100%;">}}
 
-Select **Create a New check +** in the upper right corner of the Synthetics page to create an [API tests][2] or a [browser test][3]. When a test fails, the endpoint is not retested: the uptime directly considers the endpoint as `down`.
+### Set up Private Locations
 
-{{< img src="synthetics/create_a_check.png" alt="Create a check"  style="width:80%;">}}
+Use [Synthetics Private Locations][3] to monitor internal APIs and websites - private URLs that aren’t accessible from the public internet.
 
-## Metrics
+{{< img src="synthetics/private_locations.png" alt="Private locations"  style="width:100%;">}}
 
-The following metrics are generated by Synthetics checks:
```

Comment:
Can we add a CI dedicated section, after the private location one and before the APM one?

Title: Run tests from your CI
Description: Include your Synthetic tests within your CI pipelines and start shipping without fearing faulty code might make it to production. (link to https://docs.datadoghq.com/synthetics/ci/)
Screenshot: https://a.cl.ly/mXuA6Rnx

Would it be possible to start indexing https://docs.datadoghq.com/synthetics/ci/ and add a link to reach out to support to the banner of that CI page as well?